### PR TITLE
Updating fx guidelines in docs

### PR DESCRIPTION
**Pull request by asaadmahmoodspin**
Hey @esethna have a look.

### CODE REVIEW

File: source/developer/fx-guidelines.rst [hunk 1]
```diff
@@ -1,12 +1,77 @@
-UI Elements
+FX Gudelines
```
Comment (esethna) [inline source/developer/fx-guidelines.rst]: Small typo

File: source/developer/fx-guidelines.rst [hunk 2]
```diff
@@ -1,12 +1,77 @@
...
+UI Elements
+************************
+
+|
+Header Notification
+---------------------------
+
 Appears as an additional header at the top of the UI. May include an “x” button to dismiss. Limited to 60 characters. May include a time-out. Header notification should appear over the rest of the UI instead of pushing the UI down. Multiple header header can appear, with the earliest notification at the top and more recent notifications appearing underneath.
 
 * Desktop
```
Comment (esethna) [inline source/developer/fx-guidelines.rst]: Can we make these bold rather than bullets?

**Comment by asaadmahmoodspin** [comment]
Done @esethna

**Comment by esethna** [comment]
Thanks, merged!

**Pull request merged**